README.md

```diff
@@ -1,2 +1,7 @@
-# git-brunching
-An assignment for SOFTENG701 @ UoA
+![](https://raw.githubusercontent.com/wiki/SE701Group3/git-brunching/images/Logo.png)
+
+Welcome to the 🌿 Git Brunching 🌿 repository!
+
```

Comment:
It might be helpful to add a note that says that detailed project READMEs are in the project folders.